src/routes/quantity/find.ts

```diff
@@ -40,6 +40,7 @@ interface Response {
 const controllerGeneratorOptions: ControllerGeneratorOptionsWithClientAndSupplier = {
   method: 'get',
   path: '/quantities',
+  // rights to be updated to 'Threshold' which will be added soon
```

Comment:
Make it a todo with the number of a ticket so if we ever forget we quickly can see when and where we forgot. Even better if in that ticket you also name this location because that makes the chance of forgetting much smaller

Reply:
Added the rights, no longer needed. Thanks